### PR TITLE
Add overlay and instructions for using it

### PR DESCRIPTION
**Pull request by lheckemann**

### CODE REVIEW

**Comment by dermetfan** [comment]
The `home-manager` derivation only includes the script, not the modules. When I set up home-manager [using an overlay](https://bitbucket.org/dermetfan/dotfiles/src/fad4aa6f08674df661d8ae4507607b42c7f55877/.config/nixpkgs/overlays/home-manager.nix?at=default&fileviewer=file-view-default) I had to add a source package so I can point to the modules:

```nix
self: super:

{
  home-manager-src = super.fetchFromGitHub {
    owner = "rycee";
    repo = "home-manager";
    rev = "6ecf9e091c53d592edeb202378a5b5c920dfde55";
    sha256 = "0sdmmfk2dvmn0l6k0gp6027jxlv4w4dvwd3yh8lnb5hv503xsmgb";
  };

  home-manager = import self.home-manager-src {
    pkgs = self;
    modulesPath = "${self.home-manager-src}/modules";
  };
}
```

So I  think we would have to include the modules in the `home-manager` package or create a `home-manager-modules` package or so.

File: README.md [hunk 1]
```diff
@@ -62,16 +62,11 @@ Currently the easiest way to install Home Manager is as follows:
     depending on whether you are tracking Nixpkgs unstable or version
     17.03.
 
-3.  Add Home Manager to your user's Nixpkgs, for example by adding it
-    to the `packageOverrides` section in your
-    `~/.config/nixpkgs/config.nix` file:
+3.  Add Home Manager to your user's Nixpkgs, for example by symlinking the
+    overlay to `~/.config/nixpkgs/overlays`:
 
     ```nix
```
[the remark this answers is not in the document]
Comment (lheckemann) [reply README.md]: Whoops, yes. Don't know what I was thinking there…

**Comment by rycee** [comment]
Looking good! Happy to merge to master after a squash.

**Comment by rycee** [comment]
Rebased to master in 34428fc7090744481e2328ddfda466596e875775. Thanks a lot!

**Pull request closed without merging**